modules/binary-tuple/src/main/java/org/apache/ignite/internal/binarytuple/BinaryTupleReader.java

```diff
@@ -79,6 +79,34 @@ public boolean hasNullValue(int index) {
         return begin == end;
     }
 
+    /**
+     * Reads value of specified element.
+     *
+     * @param index Element index.
+     * @return Element value.
+     */
+    public boolean booleanValue(int index) {
```

Comment:
why don't you have unit test for binary tuple?

Reply:
Added test for boolean field to BinaryTupleTest.